Increase NPTH diameter for AD-121F2 seven-segment display.

mignon-p opened the pull request:
The [datasheet](http://usasyck.com/products/AD-121F2_cat_e.pdf) says the little plastic pegs have a diameter of 1.0 mm, so I [originally](https://github.com/KiCad/Displays_7-Segment.pretty/pull/13) thought that a 1.2 mm hole would be big enough.  When I got my board back, it didn't fit.  So, I increased the hole diameter to 1.5mm and ran a test board.  It seems to fit nicely now.  Although it's a bit unsatisfying that this was determined empirically, I think it's the number we should go with.

![bigger-holes](https://user-images.githubusercontent.com/1080008/31639551-3ff85e80-b28e-11e7-8432-3f2fa22dda63.png)

mignon-p committed:
Increase NPTH diameter for AD-121F2 seven-segment display.

The datasheet says the little plastic pegs have a diameter of 1.0 mm,
so I thought that a 1.2 mm hole would be big enough.  When I got my
board back, it didn't fit.  So, I increased the hole diameter to 1.5mm
and ran a test board.  It seems to fit nicely now.  Although it's a
bit unsatisfying that this was determined empirically, I think it's
the number we should go with.

evanshultz commented:
@ppelleti 
Yeah, these things often get me too.

The datasheet also says overall tolerance is 0.2mm (these pegs themselves have a +0/-0.1mm tolerance), so we can say 1.2mm is the biggest they should get.

I think https://www.pcb-3d.com/knowledge-base/pth-dimensions/ is a good place to look for drill hole sizes. More easily accessible than the actual IPC docs, plus a free website.

So that means by Level A, which I'd suggest since this is a plastic part, we would want 1.45mm drill holes. Now it's not just empirical. If you want to change them to 1.45mm that would be great, but probably 1.5mm would also be acceptable. Let me know.

mignon-p committed:
AD-121F2: Set NPTH hole diameter to 1.45mm.

I had guessed 1.5mm and it had worked, but @evanshultz provides
convincing reasoning why 1.45mm is the "correct" value:

https://github.com/KiCad/Displays_7-Segment.pretty/pull/14

ppelleti commented:
Sure, I've gone ahead and changed it to 1.45mm on this branch.

I'll probably continue to use the 1.5mm version in my own personal projects, just because I've already validated it, and I don't think an extra 0.05mm of wiggle room is going to hurt anything.

evanshultz commented:
Thank you!
